src/components/views/game/room/RoomPlayerContainer.tsx

```diff
@@ -3,7 +3,6 @@ import { observer } from "mobx-react-lite"
 import { useCurrentGame } from "../../../../hooks/api/useCurrentGame"
 import "../../../ui/BaseContainer.scss"
 import { Tooltip } from "@mui/material"
-import BaseContainer from "../../../ui/BaseContainer"
 
 export const RoomPlayerContainer = observer(() => {
```

Comment:
I don't know if it was also horrible before, but the disabled button is not readable

![grafik](https://user-images.githubusercontent.com/1506818/170437719-2e70c662-c086-4165-9160-296885761876.png)

Reply:
This is absolutely subject to change, I was not able to custom-color the @ mui Buttons how i wanted them but @CarmenUZH managed so I will sync with her to make it work